tao_compiler/mlir/xla/ral/BUILD

```diff
@@ -200,6 +217,60 @@ cc_library(
     ],
 )
 
+# depend on mkldnn and onednn from bazel workspace
+cc_library(
+    name = "disc_mkl_bazel",
```

Comment:
renamed from `disc_mkl_bridge` but remove static mkl libs, why ....

Reply:
Originally, we don't need to depend on entire static mkl libs, only the selected ones between `start-group` and `end-group` is enough. If we depend on static mkl libs, a `multiple definitions` problems will be caused when building for torch_blade.

Reply:
Originally, we don't need to depend on entire static mkl libs, only the selected ones between `start-group` and `end-group` is enough. If we depend on static mkl libs, a `multiple definitions` problems will be caused when building for torch_blade.

Reply:
The `multiple definitions` problem seems related to whole-archive behavior in `tf_community`.
When `@mkl_static//:mkl_static_lib` is add to `deps`, these static libs will be wrapped by `whole-archive` flags in generated build scripts. While it's removed, the actual working link options are those in `linkopts`, which as no `whole-archive` flags .

Reply:
Maybe better ways is to make a cc_library for each static lib from `@mkl_static//:mkl_static_lib` instead of making all of them as one `cc_library`

Reply:
I'm trying to remove tf_communit's whole archive.